docs/prod-install.md

```diff
@@ -60,7 +60,17 @@ fail.  Also note that it automatically logs a transcript to
 `/var/log/zulip/install.log`; please include a copy of that file in
 any bug reports.
 
-## Step 3: Configure Zulip
+## Step 4: Set up email on your host
```

Comment:
I'm not a fan of combining steps here.  I think it's useful for somebody to say they're failing in step 2, without us having to dig further into which part of the step is causing them problems.